PENDING.md

```diff
@@ -42,6 +46,9 @@ tags.
 
 ### Gaia REST API
 
+* [\#3692] Update tx encoding and broadcasting endpoints:
```

Comment:
Duplicate `PENDING.md` entry?

Reply:
One was under a breaking change. I'll remove this one.